geoportal/geoportailv3_geoportal/static-ngeo/js/apps/MainController.js

```diff
@@ -903,6 +990,55 @@ const MainController = function(
     }
   }.bind(this));
 
+  listen(this.backgroundLayerMgr_, 'change', evt => {
+    const previous = evt.detail.previous;
+    const current = evt.detail.current;
+
+    // avoid if the layer is the same or first initialization
+    if (current !== previous) {
+      const label = current.get('label');
+
+      // Set accordingly the editor UI
+      if (label === 'basemap_2015_global') {
+        this.isColorVisible = true;
+        $('#editor-medium').collapse('hide');
+      } else {
+        this.isColorVisible = false;
+        $('#editor-medium').collapse('show');
+      }
+
+      // Only if current is a vector tiles layer
+      // Check if it is a function, otherwise it sould be "is not a function" error
+      if (typeof current.getMapBoxMap === 'function') {
```

Comment:
Can you have something else than `function` or `undefined` ? can't `if (current.getMapBoxMap){` be enough?